integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -866,6 +867,37 @@ case class CarbonLoadDataCommand(
           case _ =>
         }
       }
+
+      // Prepriming for Partition table here
+      val indexServerEnabled = CarbonProperties.getInstance().isDistributedPruningEnabled(
+        carbonLoadModel.getDatabaseName, carbonLoadModel.getTableName)
+      val prePrimingEnabled = CarbonProperties.getInstance().isIndexServerPrePrimingEnabled()
+      if (indexServerEnabled && prePrimingEnabled) {
+        var readCommittedScope = new TableStatusReadCommittedScope(AbsoluteTableIdentifier.from(
+          carbonLoadModel.getTablePath), hadoopConf)
+        if (carbonLoadModel.getSegmentId != null) {
+          var loadDetailsForCurrentSegment = readCommittedScope
+            .getSegmentList
+            .find(_.getLoadName.equalsIgnoreCase(carbonLoadModel.getSegmentId))
+            .get
+
+          val segmentToWrite: Segment = new Segment(carbonLoadModel.getSegmentId,
+            loadDetailsForCurrentSegment.getSegmentFile,
+            readCommittedScope,
+            loadDetailsForCurrentSegment)
+
+          val invalidSegment: List[Nothing] = List()
+          LOGGER.info(s"Adding Segments to the Cache for Partition Table")
```

Comment:
correct the log message as per old comment

Reply:
done